inc/Engine/Admin/Settings/Page.php

```diff
@@ -904,11 +865,10 @@ private function media_section() {
 		 * Lazyload Helper filter which disables WPR lazyload functionality for images.
 		 *
 		 * @since  3.4.2
-		 * @author Soponar Cristina
 		 *
 		 * @param array $disable_images_lazyload Will return the array with all plugin names which should disable LazyLoad
 		 */
-		$disable_images_lazyload = apply_filters( 'rocket_maybe_disable_lazyload_helper', $disable_images_lazyload );
+		$disable_images_lazyload = (array) apply_filters( 'rocket_maybe_disable_lazyload_helper', $disable_images_lazyload );
```

Comment:
Casting to array is not needed since `sanitize_and_format_list()`’s first test is `is_array()`.

Reply:
However, if in the future, we modify the code to do more processing with this variable, this code makes sense it's the data type we want.

The better approach is to set the data type where it has the potential to change. That way, we always know it's set to what we expect throughout our code.

Reply:
Yes, it makes sense.